src/Appwrite/Platform/Tasks/DeleteProjectCollections.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Utopia\App;
+use Utopia\Platform\Action;
+use Utopia\Cache\Cache;
+use Utopia\CLI\Console;
+use Utopia\Database\Database;
+use Utopia\Database\Query;
+use Utopia\Pools\Group;
+use Utopia\Registry\Registry;
+
+class DeleteProjectCollections extends Action
+{
+    private array $names = [
+        'webhooks',
+        'platforms',
+        'schedules',
+        'projects',
+        'domains',
+        'certificates',
+        'keys',
+        'realtime',
+    ];
+
+    public static function getName(): string
+    {
+        return 'delete-project-collections';
+    }
+
+    public function __construct()
+    {
+
+        $this
+            ->desc('Delete unnecessary project collections')
+            ->inject('pools')
+            ->inject('cache')
+            ->inject('dbForConsole')
+            ->inject('register')
+            ->callback(function (Group $pools, Cache $cache, Database $dbForConsole, Registry $register) {
+                $this->action($pools, $cache, $dbForConsole, $register);
+            });
+    }
+
+    public function action(Group $pools, Cache $cache, Database $dbForConsole, Registry $register): void
+    {
+        //docker compose exec -t appwrite delete-project-collections
+
+        Console::title('Delete project collections V1');
+        Console::success(APP_NAME . ' delete project collections');
+
+        /* Initialise new Utopia app */
+        $app = new App('UTC');
+        $console = $app->getResource('console');
+
+        /** Database connections */
+        $totalProjects = $dbForConsole->count('projects');
+        Console::success("Found a total of: {$totalProjects} projects");
+
+        $projects = [$console];
+        $count = 0;
+        $limit = 50;
+        $sum = 50;
+        $offset = 0;
```

Comment:
It might be good to expose this as a parameter in case something happens partway and we have to restart from the beginning.